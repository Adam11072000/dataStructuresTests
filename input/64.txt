init 3413
mergeSuperPixels 2110 3413 2721
setLabelScore 7519 2642 -10 5091
addImage -2
addImage -6
addImage 2418
getHighestScoredLabel 2418 2808
addImage 2418
addImage 317
getHighestScoredLabel 2418 20
getHighestScoredLabel 2418 852
getHighestScoredLabel 2418 2191
quit