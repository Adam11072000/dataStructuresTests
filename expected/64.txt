init done.
mergeSuperPixels: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
addImage: INVALID_INPUT
addImage: SUCCESS
getHighestScoredLabel: FAILURE
addImage: FAILURE
addImage: SUCCESS
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
getHighestScoredLabel: FAILURE
quit done.
